modules/nw-verify-proxy-configuration.adoc

```diff
@@ -0,0 +1,60 @@
+// Module included in the following assemblies:
+//
+// * networking/configuring-a-custom-pki.adoc
+// * networking/enable-cluster-wide-proxy.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id="nw-verify-proxy-configuration_{context}"]
+= Verifying the cluster-wide proxy configuration
+
+After the cluster-wide proxy configuration is deployed, you can verify that it is working as expected. Follow these steps to check the logs and validate the implementation.
+
+.Prerequisites
+
+* Cluster administrator permissions
+* {product-title} `oc` CLI tool installed
+
+.Procedure
+
+. Check the proxy configuration status using the `oc` command:
++
+[source,terminal]
+----
+$ oc get proxy/cluster -o yaml
+----
+
+. Verify the proxy fields in the output to ensure they match your configuration. Specifically, check the `spec.httpProxy`, `spec.httpsProxy`, `spec.noProxy`, and `spec.trustedCA` fields.
+
+. Inspect the status of the `Proxy` object:
++
+[source,terminal]
+----
+$ oc get proxy/cluster -o jsonpath='{.status}'
+----
++
+.Example output
+[source,terminal]
+----\
+{
+    "httpProxy": "http//proxy.example.com:8008",
+    "httpsProxy": https://proxy.example.com:8443",
+    "noProxy": "example.com"
```

Comment:
i think this will be much better

status:
  httpProxy: http://user:xxx@xxxx:3128
  httpsProxy: http://user:xxx@xxxx:3128
  noProxy: .cluster.local,.svc,10.0.0.0/16,10.128.0.0/14,127.0.0.1,169.254.169.254,172.30.0.0/16,localhost,test.no-proxy.com